### PR TITLE
Recipient can register for an account

### PR DESCRIPTION
**Pull request by FlisAnn**
[Recipient can register for an account](https://www.pivotaltracker.com/story/show/176623674)
### User Story
```
As an visitor,
In order to access the application,
I would like the site to have a unique way for a visitor to register
```
## Changes proposed in this pull request:
* A visitor can register for an account and become loged in and see a list of foodbags
## What I have learned working on this feature:
*
## Screenshots (Client)
Screenshot 2021-02-03 at 23 10 29

### CODE REVIEW

File: components/DisplayFoodBagsList.jsx [hunk 1]
```diff
@@ -6,16 +6,16 @@ import DisplayFoodBag from './DisplayFoodBag'
 import FoodBagService from '../modules/FoodBagService'
 
 const DisplayFoodBagsList = ({ navigation }) => {
-  const { foodbags, credentials } = useSelector(state => state)
+  const { foodbags } = useSelector((state) => state)
   useEffect(() => {
-    FoodBagService.index(credentials)
+    FoodBagService.index()
```
Comment (swardbladpaulo) [inline components/DisplayFoodBagsList.jsx:11]: Any particular reason for removing the credentials on this?

File: components/RegisterForm.jsx [hunk 2]
```diff
@@ -2,11 +2,14 @@ import React, { useState } from 'react'
...
 import { Button, StyleSheet, Text, View } from 'react-native'
 import { TextInput } from 'react-native-gesture-handler'
+import AsyncStorage from '@react-native-community/async-storage'
 
 const RegisterForm = ({ navigation }) => {
   const [email, setEmail] = useState('')
   const [password, setPassword] = useState('')
   const [passwordconfirmation, setPasswordconfirmation] = useState('')
+  const [message, setMessage] = useState()
+
```
Comment (swardbladpaulo) [inline components/RegisterForm.jsx]: Remove empty line

File: components/DisplayFoodBagsList.jsx (hunk 1 above)
Comment (FlisAnn) [reply components/DisplayFoodBagsList.jsx:11]: We are storing credentials in headers and setting them with token. Credentials actually didn't do anything

Comment (swardbladpaulo) [reply components/DisplayFoodBagsList.jsx:11]: Alright. cool

**Pull request merged**